backend/Dockerfile.backend

```diff
@@ -0,0 +1,8 @@
+FROM golang:1.22
+
+WORKDIR /app
+COPY . .
+RUN go mod download
+RUN go build -o main .
+
+CMD ["./main"]
```

Comment:
Add a newline to the end of the file please

Also minor nit, I'd prefer if the binary was just named `concinnity` as in `.gitignore`